extensions/VisualEditor/modules/ve-mw/init/targets/ve.init.mw.ViewPageTarget.js

```diff
@@ -576,9 +576,29 @@ ve.init.mw.ViewPageTarget.prototype.onSaveErrorNewUser = function ( isAnon ) {
  * @method
  */
 ve.init.mw.ViewPageTarget.prototype.onSaveErrorCaptcha = function () {
-	// Wikia change: Only support reCAPTCHA
+	// Wikia change: Only support reCAPTCHA and FancyCaptcha
 	this.captchaResponse = null;
 	this.saveDialog.$captcha.empty();
+
+	// reCaptcha loaded without any problems
+	if (this.saveDialog.frame.$element[0].contentWindow.grecaptcha !== undefined) {
```

Comment:
Captcha.js takes care of loading FancyCaptcha and swapping it out in the DOM for all captchas outside of VE. If reCaptcha fails to load when inside of VE however, `this.saveDialog.frame.$element[0].contentWindow.grecaptcha` is undefined and an error will be thrown here. Additionally, Captcha.js expects the reCaptcha form to have been injected into the page when falling back to FancyCaptcha. This isn't the case for VE so some additional logic had to be added.